testsuite/integration/manualmode/pom.xml

```diff
@@ -46,6 +46,35 @@
             -->
             <build>
                 <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>1.6.0</version>
+                        <executions>
+                            <execution>
+                                <id>generate-jboss-stores</id>
+                                <phase>test-compile</phase>
+                                <goals>
+                                    <goal>java</goal>
+                                </goals>
+                                <configuration>
+                                    <mainClass>org.jboss.as.test.manualmode.ejb.ssl.GenerateJBossStores</mainClass>
+                                    <classpathScope>test</classpathScope>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>generate-ldap-stores</id>
```

Comment:
```generate-ldap-stores``` should no longer be needed since the ldap tests that rely on these generated files now call ```GenerateLdapConnectionStores.setUpKeyStores()``` in ```@BeforeClass``` methods.

Reply:
Fixed